web-ng/src/app/components/feature-panel/feature-panel.component.ts

```diff
@@ -72,6 +83,20 @@ export class FeaturePanelComponent {
       );
   }
 
+  ngOnInit() {
+    this.subscription.add(
+      this.navigationService.getProjectId$().subscribe(id => {
+        this.projectId = id;
```

Comment:
I think you can convert from `string | undefined | null` to `string | undefined` by replacing `id` with `id || undefined`.